Bottom: 254e17ef0751ce8ba10eb296caa91caaf9e1d428
Top:    9e5b37515aa889b7537016a8961e4fed2f021d41
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-13 15:45:42 -0500

Refresh of production

---
diff --git a/provision.sh b/provision.sh
index 3762e89d..d58ff0fa 100755
--- a/provision.sh
+++ b/provision.sh
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 #PYTHONUNBUFFERED=1 \
-     
+
 ANSIBLE_ENABLE_TASK_DEBUGGER=True \
 ANSIBLE_FORCE_COLOR=true \
 ANSIBLE_HOST_KEY_CHECKING=false \
@@ -9,6 +9,6 @@ ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o Contr
 exec ansible-playbook \
         --connection=ssh \
         --timeout=30 \
-        --inventory-file=ansible-playbook/staging \
+        --inventory-file=ansible-playbook/production \
         -vv \
         ansible-playbook/site.yml "$@"
